scripts/base_network.py

```diff
@@ -317,8 +378,16 @@ def _rebase_voltage_to_config(component):
 
 def base_network():
     buses = _load_buses_from_osm().reset_index()
+
     lines = _load_lines_from_osm(buses)
+    lines_dc = _load_links_from_osm(buses)
+
     lines = _set_electrical_parameters_lines(lines)
```

Comment:
Should this electrical parameters be applied on lines_ac_dc? Rather than removing this line I would comment it and duplicated after adaptation below

Reply:
Thank you! Corrected.